config/schedule.rb

```diff
@@ -13,6 +13,8 @@
 # If changing the schedule of one of these jobs, also update at https://app.honeybadger.io/projects/54415/check_ins
 job_type :rake_rb, 'cd :path && :environment_variable=:environment bundle exec rake --silent ":task" :output && curl --silent https://api.honeybadger.io/v1/check_in/:check_in'
 job_type :runner_hb, 'cd :path && bin/rails runner -e :environment ":task" :output && curl --silent https://api.honeybadger.io/v1/check_in/:check_in'
+# Overriding default runner job_type to remove invoking bundle exec.
+job_type :runner, 'cd :path && bin/rails runner -e :environment ":task" :output'
```

Comment:
Perusing https://github.com/javan/whenever/issues/732 and the whenever code, the other option would be to use:

```ruby
set :runner_command, 'rails runner'
```

and let whenever handle the rest. I'm not sure that's better than redefined the `:runner` job type, though? 🤷🏻‍♂️

Reply:
I'd suggest that the more explicit specification makes it clear what the `runner` is invoking (without having to look at whenever code).